Sources/Sentry/SentryScope.m

```diff
@@ -531,7 +531,7 @@ - (SentryEvent *__nullable)applyToEvent:(SentryEvent *)event
                 [span isKindOfClass:[SentryTracer class]]) {
                 event.transaction = [[(SentryTracer *)span transactionContext] name];
             }
-            newContext[@"trace"] = [span.context serialize];
+            newContext[@"trace"] = [span serialize];
```

Comment:
`m`: Isn't this adding more information than previously? Is that acceptable?

Reply:
good point. Probably should keep the same amount of information

Reply:
According to [docs](https://develop.sentry.dev/sdk/event-payloads/transaction/#contextstrace), looks like this is the right thing to do.

Reply:
Does it still make sense to pass the type as `SentrySpanContext` in SentrySpan.serialize line 161, or should that also be `SentrySpan`? https://github.com/getsentry/sentry-cocoa/blob/55b4922f5b20549a939ace0faed62abcc6a4814a/Sources/Sentry/SentrySpan.m#L158-L166

Reply:
`SentrySpanContext.type` is a static property that returns "trace".
I have no idea why I made this way, we probably should use a const here.

Reply:
> A Transaction has to have a specific contexts.trace entry that contains data from the [Span](https://develop.sentry.dev/sdk/event-payloads/span/).

Yes, data from the span. Really everything? The whole span?